### DISCUSSION TITLE
how to build amazon-k8s-cni-init

### DISCUSSION THREAD

**Discussion by sandeep-robin** [aws/amazon-vpc-cni-k8s#1924] [Q&A] [answered]
Hi,

i want to build amazon-k8s-cni and amazon-k8s-cni-init docker images. make docker-build builds  **amazon-k8s-cni** docker image.
how about **amazon-k8s-cni-init**. ? 

or 

is there any way i can pull these docker images directly .? aws-k8s-cni.yaml has image location as 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.10.2.  i cannot pull this image.

**Comment by jayanthvn** [comment, marked as answer]
Hi, you can build both images by running `make docker && make docker-init` [https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L117-L130] or `make all` [https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L99]
